### PR TITLE
test: update to tap@1.3.4

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

**Comment by rmg** [comment]
LGTM.

File: test/test-watch-cpu-profile.js [hunk 1]
```diff
@@ -85,11 +87,13 @@ tap.test('cpu-profile', skipIfNotLinux || function(t) {
         tt.equal(msg.wid, worker.id);
       } else {
         tt.equal(msg.cmd, 'cpu-profiling');
-        tt.equal(msg.id, worker.id);
+        tt.equal(msg.wid, worker.id);
```
Comment (sam-github) [inline test/test-watch-cpu-profile.js]: @rmg no idea why this is showing up now

**Comment by sam-github** [comment]
I'll merge when green

**Comment by sam-github** [comment]
@slnode test please

**Pull request merged**